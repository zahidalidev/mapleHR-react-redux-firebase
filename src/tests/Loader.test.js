import { shallow } from 'enzyme'

import Loader from 'components/loader'

describe('loader', () => {
  test('renders', () => {
    expect(shallow(<Loader show={true} />).exists()).toBe(true)
  })

  test('loader should return null', () => {
    expect(shallow(<Loader show={false} />).isEmptyRender()).toBe(true)
  })

  test('should render loader', () => {
    expect(shallow(<Loader show={true} />).getElements()).toMatchSnapshot()
  })
})
