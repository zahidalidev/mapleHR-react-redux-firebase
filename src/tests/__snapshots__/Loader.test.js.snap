// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loader should render loader 1`] = `
Array [
  <div
    className="backdrop"
  >
    <div
      className="loading-spinner-wrapper"
    >
      <div
        className="loading-spinner"
      >
        <div />
      </div>
    </div>
  </div>,
]
`;
